### DISCUSSION TITLE
IOS 16.6 部分动画不显示

### DISCUSSION THREAD

**Discussion by kieranwv** [Tencent/libpag#2610] [General]
有的动画好使，有的不好使。

![image](https://github.com/user-attachments/assets/ef578093-7b86-4ce9-a4ed-213f97b39603)

![a337507b42f5c6fc53110c40cb3af2f](https://github.com/user-attachments/assets/aa7f5dea-b8a2-4c79-a68d-e5d3109daf52)

**Comment by kevingpqi123** [comment]
基于 https://pag.io/player.html 这个页面预览相关效果，如果效果存在缺失，将 pag 文件附上

**Comment by jinwuwu001** [comment]
该问题长时间未回复，先关闭，若有问题可重新提
